hazelcast-jet-core/src/main/java/com/hazelcast/jet/JetInstance.java

```diff
@@ -134,6 +138,39 @@ default Job getJob(@Nonnull String name) {
         return getJobs(name).stream().findFirst().orElse(null);
     }
 
+    /**
+     * Returns the IMap that contains the data of an exported snapshot with the
+     * given name.
+     * <p>
+     * The contents of the map are unspecified.
+     *
+     * @param name exported snapshot name
+     * @return IMap instance with the data
+     */
+    default IMapJet<Object, Object> getExportedState(@Nonnull String name) {
```

Comment:
it shouldn't just return a map. The underlying map should not be visible to the user. Instead some wrapper object which has some additional info would be more useful. i.e. when the snapshot was taken, which job it originated from, how big is it.

Reply:
fixed, now returns `JobStateSnapshot`